src/notebook/Cell.jl

```diff
@@ -27,6 +27,11 @@ Base.@kwdef mutable struct Cell
     module_usings::Set{Expr}=Set{Expr}()
     rootassignee::Union{Nothing,Symbol}=nothing
     function_wrapped::Bool=false
+
+    # information to display cell dependencies
+    cell_execution_order:: Union{Missing, Int} = missing
+    referenced_cells:: Union{Missing, Vector{Int}} = missing
+    dependent_cells:: Union{Missing, Vector{Int}} = missing
```

Comment:
I added cell order (as a list of all cell UUIDs) as notebook property, but kept the order of each cell (as integer) in as cell property for now. If this causes issues, it could be removed later.